src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditCommand.java

```diff
@@ -14,10 +14,17 @@
 
     public static final String COMMAND_WORD = "edit";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Edits the person identified by the index number used in the displayed item list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+    public static final String SPECIFY_INDEX = "Please Specify the index";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " with optional fields\n"
+            + "Edits a travel plan to the travel planner using the format:\n"
+            + "edit -travelplan n/NAME sd/START_DATE ed/END_DATE\n"
+            + "Edits an activity to the current travel plan using the format:\n"
+            + "edit -activity n/NAME i/LEVEL_OF_IMPORTANCE l/LOCATION c/COST d/DATE_AND_TIME\n"
+            + "Edits an accommodation to the current travel plan using the format:\n"
+            + "edit -accommodation n/NAME l/LOCATION c/COST n/NIGHTS\n"
```

Comment:
I think it is suppose to be start date and end date?

Reply:
yes, I will change it. thankss